clang/include/clang/Sema/Sema.h

```diff
@@ -1759,6 +1759,33 @@ class Sema final {
     }
   };
 
+  /// Bitmask to contain the list of reasons a single diagnostic should be
+  /// emitted, based on its language.  This permits multiple offload systems
+  /// to coexist in the same translation unit.
+  enum DeviceDiagnosticReason {
+    /// Diagnostic doesn't apply to anything. Included for completeness, but
+    /// should make this a no-op.
+    DDR_None = 0,
+    /// OpenMP specific diagnostic.
+    DDR_OMP_DEVICE = 1 << 0,
```

Comment:
The enumerator names look more like macro names, which isn't prevalent in the rest of the code base. I'd suggest using the usual naming conventions.

Reply:
Fixed.